cylc/rose/utilities.py

```diff
@@ -192,13 +192,38 @@ def identify_templating_section(config_node):
             "You should not define more than one templating section. "
             f"You defined:\n\t{'; '.join(defined_sections)}"
         )
-    elif 'jinja2:suite.rc' in defined_sections:
-        templating = 'jinja2:suite.rc'
-    elif 'empy:suite.rc' in defined_sections:
-        templating = 'empy:suite.rc'
     else:
+        return _identify_templating_section(defined_sections)
+
+
+def _identify_templating_section(
```

Comment:
This shouldn't have an underscore prefix.

Underscore prefix means, don't use this outside of the scope it's defined in e.g:

* Underscore-prefixed method: only the class should use this
* Underscore-prefixed function: only the module should use this
* Underscore-prefixed module: only the package should use this